package.json

```diff
@@ -259,19 +259,19 @@
     "lint-autofix": "eslint --fix \"{compatibility,example,features,scripts,src,test}/**/*.ts\"",
     "lint-code": "eslint \"{compatibility,example,features,scripts,src,test}/**/*.ts\"",
     "lint-dependencies": "dependency-lint",
-    "lint": "yarn run lint-code && yarn run lint-dependencies",
-    "prelint-autofix": "yarn build-local",
-    "prelint-code": "yarn build-local",
-    "precck-test": "yarn run build-local",
-    "prefeature-test": "yarn run build-local",
+    "lint": "npm run lint-code && npm run lint-dependencies",
+    "prelint-autofix": "npm run build-local",
+    "prelint-code": "npm run build-local",
+    "precck-test": "npm run build-local",
+    "prefeature-test": "npm run build-local",
     "prepublishOnly": "rm -rf lib && npm run build-local",
-    "pretest-coverage": "yarn build-local",
-    "pretypes-test": "yarn build-local",
+    "pretest-coverage": "npm run build-local",
+    "pretypes-test": "npm run build-local",
     "test-coverage": "nyc --silent mocha 'src/**/*_spec.ts' 'compatibility/**/*_spec.ts' && nyc --silent --no-clean node ./bin/cucumber-js && nyc report --reporter=lcov",
-    "test": "yarn run lint && yarn run types-test && yarn run unit-test && yarn run cck-test && yarn run feature-test",
+    "test": "npm run lint && npm run types-test && npm run unit-test && npm run cck-test && npm run feature-test",
     "types-test": "tsd",
     "unit-test": "mocha 'src/**/*_spec.ts'",
-    "update-dependencies": "npx npm-check-updates --upgrade"
+    "update-dependencies": "npx npm run-check-updates --upgrade"
```

Comment:
This doesn't look quite right - does it work?

Reply:
@davidjgoss, test the change did not work so I reverted back the changes.I should have the individual commands but did not so sorry. 
Abel

Reply:
No problem, thanks!